dotcom-rendering/src/web/components/Island.tsx

```diff
@@ -44,7 +42,7 @@ const decideChildren = (
 	placeholderHeight?: number,
 ) => {
 	if (!clientOnly) return children; // Server side rendering
-	if (placeholderHeight)
+	if (placeholderHeight !== undefined && placeholderHeight !== 0)
```

Comment:
I don’t think a negative of 0 value make any sense, here.

```suggestion
	if (placeholderHeight !== undefined && placeholderHeight > 0)
```